Django 1.8 DurationField not supported by make

When running `model_mommy.mommy.make` on a model containing a `DurationField`, I get the following error:

```
E           TypeError: <class 'django.db.models.fields.DurationField'> is not supported by mommy.
```

--- a/model_mommy/generators.py
+++ b/model_mommy/generators.py
@@ -148,6 +148,12 @@ def gen_byte_string(max_length=16):
     elif six.PY3:
         return bytes(generator)
 
+def gen_interval(interval_key='milliseconds'):
+    from datetime import timedelta
+    interval = gen_integer()
+    kwargs = {interval_key: interval}
+    return timedelta(**kwargs)
+
 def gen_content_type():
     from django.contrib.contenttypes.models import ContentType
     from django.db.models import get_models

--- a/model_mommy/mommy.py
+++ b/model_mommy/mommy.py
@@ -39,6 +39,11 @@ try:
 except ImportError:
     BinaryField = None
 
+try:
+    from django.db.models import DurationField
+except ImportError:
+    DurationField = None
+
 from django.core.exceptions import ValidationError
 from django.core.validators import validate_ipv4_address
 try:
@@ -158,6 +163,8 @@ default_mapping = {
 
 if BinaryField:
     default_mapping[BinaryField] = generators.gen_byte_string
+if DurationField:
+    default_mapping[DurationField] = generators.gen_interval
 
 
 class ModelFinder(object):

--- a/test/generic/models.py
+++ b/test/generic/models.py
@@ -84,6 +84,11 @@ class Person(models.Model):
     except AttributeError:
         wanted_games_qtd = models.IntegerField()
 
+    try:
+        duration_of_sleep = models.DurationField()
+    except AttributeError:
+        pass
+
 
 class Dog(models.Model):
     owner = models.ForeignKey('Person')

--- a/test/generic/tests/test_filling_fields.py
+++ b/test/generic/tests/test_filling_fields.py
@@ -1,4 +1,4 @@
-from datetime import date, datetime, time
+from datetime import date, datetime, time, timedelta
 from decimal import Decimal
 from os.path import abspath
 from tempfile import gettempdir
@@ -46,6 +46,11 @@ try:
 except ImportError:
     BinaryField = None
 
+try:
+    from django.db.models.fields import DurationField
+except ImportError:
+    DurationField = None
+
 from django.core.validators import validate_ipv4_address
 try:
     from django.core.validators import validate_ipv6_address, validate_ipv46_address
@@ -65,6 +70,9 @@ __all__ = [
 
 if BinaryField:
     __all__.append('BinaryFieldsFilling')
+
+if DurationField:
+    __all__.append('DurationField')
 
 
 def assert_not_raise(method, parameters, exception):
@@ -125,6 +133,15 @@ class BinaryFieldsFilling(FieldFillingTestCase):
             name_hash = self.person.name_hash
             self.assertIsInstance(name_hash, binary_type)
             self.assertEqual(len(name_hash), name_hash_field.max_length)
+
+
+class DurationFieldsFilling(FieldFillingTestCase):
+    if DurationField:
+        def test_fill_DurationField_with_random_interval_in_miliseconds(self):
+            duration_of_sleep_field = Person._meta.get_field('duration_of_sleep')
+            self.assertIsInstance(duration_of_sleep_field, DurationField)
+            duration_of_sleep = self.person.duration_of_sleep
+            self.assertIsInstance(duration_of_sleep, timedelta)
 
 
 class BooleanFieldsFilling(FieldFillingTestCase):

